easybuild/easyconfigs/a/AtomPAW/AtomPAW-4.1.0.5-intel-2018b.eb

```diff
@@ -0,0 +1,39 @@
+##
+# This version of AtomPAW is lastest version to be used with ABINIT 8.10.x
```

Comment:
if it's a problem with `AtomPAW` 4.1.0.6, wouldn't it be better to add this comment in that easyconfig, or on the `ABINIT` easyconfig?

If a later version of `AtomPAW` can be used with `ABINIT`, this comment will be outdated...

Reply:
Thank you for this comment....
I was indeed very annoyed... :-(

To clarify the situation:

1. AtompPAW version 4.1.0.6 has no intrinsic problems
2. ABINIT version 8.10.3 is the latest in the 8.10 series
3. The future production version of ABINIT will be a version 9.0 which is compatible with AtomPAW 4.1.0.6
4. We could even remove the sentence in the AtomPAW 4.1.0.5 config file : 

`# This version of AtomPAW is lastest version to be used with ABINIT 8.10.x`

Reply:
ok, then the sentence seems correct, although I'd say it's more important to state that in the `ABINIT ` easyconfig than here